tests.test.js

```diff
@@ -0,0 +1,106 @@
+const app = require(`./main.js`);
+const supertest = require('supertest');
+const request = supertest(app);
+const { test, expect } = require("@jest/globals");
+const fs = require("fs");
```

Comment:
You should use your DB class to read / write to your test DB. There is no need to use `fs` here unless you have a really good reason.